pkg/bpf/bpf_restart.go

```diff
@@ -53,6 +53,7 @@ const (
 )
 
 var kmeshStartType StartType
+var kmeshExitType StartType
```

Comment:
Add comments about when to use `kmeshExitType` and when to use `kmeshStartType`, it is also very confusing that they are both of type `StartType`, maybe we need better naming.

Reply:
kmeshStartType is used during bootsttap

kmeshExitType  is used during exit, to indicate whether keep bpf prog/maps or not

Previously one var is used to indicate both, confusing to me